supervision/tracker/ByteTrack/basetrack.py

```diff
@@ -0,0 +1,50 @@
+from collections import OrderedDict
+import numpy as np
+
+class TrackState(object):
```

Comment:
That can be Python `Enum`. Take a look at the `Position` we already have in our codebase.